ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Backgrounds/NewTabPageVideoButtonsView.swift

```diff
@@ -73,14 +73,10 @@ class NewTabPageVideoButtonsView: UIView {
 
     if playStarted {
       playPauseButtonImage.imageView.image = UIImage(braveSystemNamed: "leo.play.circle")!
-        .applyingSymbolConfiguration(
-          .init(scale: .large)
-        )
+        .withAlignmentRectInsets(.zero)
     } else {
       playPauseButtonImage.imageView.image = UIImage(braveSystemNamed: "leo.pause.circle")!
-        .applyingSymbolConfiguration(
-          .init(scale: .large)
-        )
+        .withAlignmentRectInsets(.zero)
```

Comment:
`withAlignmentRectInsets(.zero)` was needed to properly align SF pause symbol:

![image](https://github.com/brave/brave-core/assets/86598290/31bc0809-bb87-43d7-8c74-9850133236f2)